lib/routes/home_route/tabs_route.dart

```diff
@@ -386,145 +410,156 @@ class _ContentTabState extends State<_ContentTab> with AutomaticKeepAliveClientM
   @override
   Widget build(BuildContext context) {
     super.build(context);
+    final l10n = getl10n(context);
     final theme = ThemeControl.instance.theme;
     final contentType = widget.contentType;
-    final list = ContentControl.instance.getContent(contentType);
-    final showDisabledActions = list.isNotEmpty && list.first is Playlist && (list as List<Playlist>).every((el) => el.songIds.isEmpty);
-    final selectionController = widget.selectionController;
-    final selectionRoute = selectionRouteOf(context);
-    return RefreshIndicator(
-      key: key,
-      strokeWidth: 2.5,
-      color: theme.colorScheme.onPrimary,
-      backgroundColor: theme.colorScheme.primary,
-      onRefresh: ContentControl.instance.refetchAll,
-      notificationPredicate: (notification) {
-        return selectionController.notInSelection &&
-               notification.depth == 0;
-      },
-      child: ContentListView(
-        contentType: contentType,
-        list: list,
-        showScrollbarLabel: showLabel,
-        selectionController: selectionController,
-        onItemTap: contentPick<Content, ValueSetter<int>>(
+    return ValueListenableBuilder<bool>(
+      valueListenable: FavoritesControl.instance.onFavoritesMode,
+      builder: (context, favoriteMode, child) {
+        final list = ContentControl.instance.getContent(
           contentType: contentType,
-          song: (index) => QueueControl.instance.resetQueue(),
-          album: (index) {},
-          playlist: (index) {},
-          artist: (index) {},
-        ),
-        leading: Column(
-          children: [
-            if (selectionRoute)
-              ContentListHeader.onlyCount(contentType: contentType, count: list.length)
-            else
-              ContentListHeader(
+          filterFavorite: favoriteMode,
+        );
+        final showDisabledActions = list.isNotEmpty && list.first is Playlist && (list as List<Playlist>).every((el) => el.songIds.isEmpty);
+        final selectionController = widget.selectionController;
+        final selectionRoute = selectionRouteOf(context);
+        return RefreshIndicator(
+          key: refreshIndicatorKey,
+          strokeWidth: 2.5,
+          color: theme.colorScheme.onPrimary,
+          backgroundColor: theme.colorScheme.primary,
+          onRefresh: ContentControl.instance.refetchAll,
+          notificationPredicate: (notification) {
+            return selectionController.notInSelection &&
+                  notification.depth == 0;
+          },
+          child: favoriteMode && list.isEmpty
```

Comment:
The list here cannot be empty under any condition not in `favoriteMode` because:

* If there are no songs, we don't show the home route
* If there are songs, there is at least one album and at least one artist
* If there are no playlists, this seems redundant to show a message about it, seems we have a button to create a playlist and the "0 playlists" text at the top of the screen

This condition just manifests when what codepath this is handling, since it specifically doens't aim to show this message under any other conditions, other than in favorites mode.